### PR TITLE
[docs] FCM-specific advice for InvalidCredentials error

### PR DESCRIPTION
**Pull request by keith-kurak**
# Why

The `InvalidCredentials` error can happen for FCM/ Android, as well, but we only had iOS-specific advice, which also previously referenced Classic build commands.

# How

With FCM, `InvalidCredentials` can be returned on a push receipt [due to a 401 from the FCM endpoints](https://github.com/expo/universe/blob/main/server/push-hydrant/src/fcm/FcmSend.ts#L86), which [seems to be associated with passing an incorrect server key](https://stackoverflow.com/questions/37633188/firebase-401-unauthorized-error-fcm), which we address in our setup instructions, so the the troubleshooting advice links back to that. Would welcome any additional suggestions for Android/ FCM troubleshooting.

# Test Plan

image

### CODE REVIEW

File: docs/pages/push-notifications/sending-notifications.mdx [hunk 1]
```diff
@@ -209,15 +209,15 @@ Inside both push tickets and push receipts, look for a `details` object with an
...
 
-- `InvalidCredentials`: your push notification credentials for your standalone app are invalid (ex: you may have revoked them). Run `expo build:ios -c` to regenerate new push notification credentials for iOS. If you revoke an APN key, all apps that rely on that key will no longer be able to send or receive push notifications until you upload a new key to replace it. Uploading a new APN key will **not** change your users' Expo Push Tokens.
-  - Sometimes, these errors will contain further details claiming an `InvalidProviderToken` error. This is actually tied to both your APN key **and** your provisioning profile. To resolve this error, you should rebuild the app and regenerate a new push key and provisioning profile.
-
+- `InvalidCredentials`: Your push notification credentials for your standalone app are invalid (ex: you may have revoked them).
+  - **Android**: Be sure that you have correctly applied the server key from the Firebase Console as specified in [the instructions for uploading FCM server credentials](using-fcm/#uploading-server-credentials).
```
Comment (amandeepmittal) [inline docs/pages/push-notifications/sending-notifications.mdx:219]:
```suggestion
  - **Android**: Make sure that you have correctly uploaded the server key from the Firebase Console as specified in the [uploading FCM server credentials](using-fcm/#uploading-server-credentials).
```

**Review by Simek — APPROVED** [review]
LGTM! 👍

File: docs/pages/push-notifications/sending-notifications.mdx [hunk 2]
```diff
@@ -209,15 +209,15 @@ Inside both push tickets and push receipts, look for a `details` object with an
...
+  - **iOS**: Run `eas credentials` and follow the prompts to regenerate new push notification credentials. If you revoke an APN key, all apps that rely on that key will no longer be able to send or receive push notifications until you upload a new key to replace it. Uploading a new APN key will **not** change your users' Expo Push Tokens. Sometimes, these errors will contain further details claiming an `InvalidProviderToken` error. This is actually tied to both your APN key **and** your provisioning profile. To resolve this error, you should rebuild the app and regenerate a new push key and provisioning profile.
 > For a better understanding of iOS credentials, including push notification credentials, read our [App Signing docs](/app-signing/app-credentials.mdx#ios)
```
Comment (amandeepmittal) [inline docs/pages/push-notifications/sending-notifications.mdx:221]:
```suggestion
> For a better understanding of iOS credentials, including push notification credentials, read our [App Signing docs](/app-signing/app-credentials.mdx#ios).
```

Unrelated to the changes in this PR. Let's add the missing period here :D

**Review by amandeepmittal — APPROVED** [review]
Thanks for updating it and adding more clarification 👍 

Left some small suggestions :D

**Pull request merged**